File: src/main/java/org/ventanas/Java/Ejercicio6.java
package org.ventanas.Java;
// 6. Sumar números del 1 al 100
// Crear un programa que sume todos los números del 1 al 100.
public class Ejercicio6 {
    public static void execute() {
        int cont = 0;
        for (int i = 1; i <= 100; i++) {
            int newCont = cont + i;
            System.out.println(cont + " + " + i + " = " + newCont);
            cont = newCont;
        }
    }
}

File: src/main/java/org/ventanas/Kotlin/Ejercicio6.kt
package org.ventanas.Kotlin
// 6. Sumar números del 1 al 100
// Crear un programa que sume todos los números del 1 al 100.

fun ejercicio6() {
    var cont = 0
    for (i in 1..100){
        val newCont = cont +i
        println("$cont + $i = ${newCont}")
        cont = newCont
    }
}